lib/ts/recipe/totp/recipe.tsx

```diff
@@ -71,10 +70,6 @@ export default class TOTP extends RecipeModule<
             if (mfa !== undefined) {
                 mfa.addMFAFactors([totpFactor]);
             }
-            const session = Session.getInstance();
-            if (session !== undefined) {
-                session.addAuthRecipeRedirectionHandler("totp", this.redirect.bind(this));
-            }
```

Comment:
why is this removed?

Reply:
We do not need it anymore because the success redirection is handled differently